.github/workflows/pr_checks.yaml

```diff
@@ -1,27 +1,40 @@
-name: Frontend PR Checks
-on: # see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows
-  workflow_dispatch:
-  pull_request:
-    paths:
-      - "src/**/*"
-      - "package.json"
-      - "package-lock.json"
-      - "*.js"
+name: PR Checks
+
+on:
+  push:
+    branches:
+      - staging
+
 jobs:
-  lint_and_build:
+  lint:
     runs-on: ubuntu-latest
+
     steps:
-      - name: Check out repository code
-        uses: actions/checkout@v4
-      - name: Setup Node.js
-        uses: actions/setup-node@v4
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
Hi @nlebovits why is the PR downgrading `actions/checkout` from v4 to v3?

Reply:
@CodeWritingCow good catch! sloppy chatgpt hallucinations lol

Reply:
will fix in a new commit + check the other PR for backend work to do the same

Reply:
@CodeWritingCow going to add a `dependabot.yml` to keep these up to date. see #757